### PR TITLE
Update Trust Tokens post to include DevTools information

### PR DESCRIPTION
**Pull request by samdutton**
This PR also moves images to the _.../trust-tokens/images_ directory.

@jpmedley @chybie fyi — tweaks/corrections/additions welcome.

### CODE REVIEW

File: src/site/content/en/blog/trust-tokens/index.md [hunk 1]
```diff
@@ -27,7 +27,21 @@ in other contexts to evaluate the user's authenticity.
 The Trust Token API enables trust of a user in one context to be conveyed to another context without
 identifying the user or linking the two identities.
 
-You can try out Trust Tokens with our [demo](https://trust-token-demo.glitch.me).
+You can try out Trust Tokens with our [demo](https://trust-token-demo.glitch.me) and, from Chrome 
+89, [inspect Trust Tokens](https://developers.google.com/web/updates/2021/01/devtools#trust-token) 
+in the Chrome DevTools Network and Application tabs.
```
Comment (mihajlija) [inline src/site/content/en/blog/trust-tokens/index.md:32]:
```suggestion
in the Chrome DevTools **Network** and **Application** tabs.
```

File: src/site/content/en/blog/trust-tokens/index.md [hunk 2]
```diff
@@ -27,7 +27,21 @@ in other contexts to evaluate the user's authenticity.
...
+
+<figure class="w-figure">
+  <img src="./images/devtools-network-tab.png" alt="Screenshot showing Trust Tokens in the Chrome 
+  DevTools Network tab.">
+  <figcaption class="w-figcaption">Trust Tokens in the Chrome DevTools <b>Network</b> tab</figcaption>
```
Comment (mihajlija) [inline src/site/content/en/blog/trust-tokens/index.md:37]:
```suggestion
  <figcaption class="w-figcaption">Trust Tokens in the Chrome DevTools <b>Network</b> tab.</figcaption>
```

File: src/site/content/en/blog/trust-tokens/index.md [hunk 3]
```diff
@@ -27,7 +27,21 @@ in other contexts to evaluate the user's authenticity.
...
+</figure>
+
+<figure class="w-figure">
+  <img src="./images/devtools-application-tab.png" alt="Screenshot showing Trust Tokens in the Chrome 
+  DevTools Application tab.">
+  <figcaption class="w-figcaption">Trust Tokens in the Chrome DevTools <b>Application</b> tab</figcaption>
```
Comment (mihajlija) [inline src/site/content/en/blog/trust-tokens/index.md:43]:
```suggestion
  <figcaption class="w-figcaption">Trust Tokens in the Chrome DevTools <b>Application</b> tab.</figcaption>
```

**Comment by samdutton** [comment]
Thanks @mihajlija!

**Review by mihajlija — APPROVED** [review]
LGTM, I'll just suggest slight image optimization to keep our repo size down. I tried switching to MozJPEG in squoosh and it gives ~50% size reduction with similar image quality.

**Comment by samdutton** [comment]
> I tried switching to MozJPEG in squoosh and it gives ~50% size reduction with similar image quality.

I see what you mean — the Photoshop JPEG wasn't really worth it, but MozJPEG is almost as good as the PNG.

I've updated the images and index.md.

**Comment by kaycebasques** [comment]
Hello! This is an automated review. It updates automatically when code or GitHub comments in this pull request are created or updated.

## Requested changes

If there are any common problems with the content files you created or modified, they will be listed here.

### `src/site/content/en/blog/trust-tokens/index.md`

* This file passed all of our automated Markdown audits.

**Comment by chybie** [comment]
LGTM! Thanks @samdutton. :D

**Pull request merged**